### PR TITLE
Remove unused files, add configuration guide for Issuer Kit services.

### PR DESCRIPTION
**Pull request by esune**
Added configuration guide for Issuer Kit services.

The files that have been removed are unused.

### CODE REVIEW

File: docs/ConfiguringIssuerKit.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+The contents of the file will be displayed below the generic unauthorized message.
+
+#### Home page
+
+Similarly to the `unauthorized.html` configuration file we discussed earlier, `terms-and-confitions.html` defines custom HTML/CSS that will be injected in the main landing page for the issuer webapp. This can be used to add a welcome message or a disclaimer.
+
+#### Dynamic Form Configuration
+
+Both the admin and issuer webapp use SurveyJS to display forms that are dynamically generated from a configuration file.
```
Comment (swcurran) [inline docs/ConfiguringIssuerKit.md:35]: Please add link to `[SurveyJS](https://surveyjs.io)`.  Nice to note the connection between the config file referenced here and the one below.  Maybe change this to "a SurveyJS configuration file, referenced below."

File: docs/ConfiguringIssuerKit.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+`config/claim-config.json` is  SurveyJs configuration file, defining the behaviour of the surveyjs object. For more details on how to configure a survey, please refer to the [SurveyJS docs](https://surveyjs.io/Examples/Library).
+
+`config/custom-scripts.js` is used to define custom JS scripts that can be used in the SurveyJS form to customize its behaviour using triggers. All functions need to be declared in the file, and added to the `surveyFunctions` array in order to be registered with the SurveyJS component when it is rendered.
+
+#### Application configuration
+
+`config.json` contains the bulk of the configuration for the web application. Most of the settings in this file are shared between the admin and issuer wbapps, with some exceptions.
+
+* `env`: a string describing the environment the deployment is running in.
```
Comment (swcurran) [inline docs/ConfiguringIssuerKit.md:45]: Clarification on what that is/means?  Is it just a label?

File: docs/ConfiguringIssuerKit.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+#### Main configuration file
+
+Most of the configuration file is fairly simple, the following are the setting groups that require some additional details:
+
+* `smtp`: this is a Nodemailer configuration that gets passed as-is to the underlying nodemailer plugin. For a reference of the available settings, please see the [Nodemailer docs](https://nodemailer.com/smtp/).
+
+* `emailSettings`: this object contains configuration items used when generating and sending the invite email. `sender` and `subject` are required, but additional properties can be defined and used in the invite email (see details in the invite email section).
+
+* `publisSite`: the base URl for the public issuer site, to be used when generating invite links.
```
Comment (swcurran) [inline docs/ConfiguringIssuerKit.md:86]: Typo in the name "publis"?

File: docs/ConfiguringIssuerKit.md (hunk 2 above)
Comment (esune) [reply docs/ConfiguringIssuerKit.md]: Added additional details. This is basically a label that I use in the config file to quickly figure out whether its the file for dev/test/prod.

**Comment by esune** [comment]
Yes, the way the `demo` mode works - as an example - is by building the docker images for each one of the services and starting them with overrides for one (or more) of the configuration files.

I think the docs highlight where the overrides need to be placed, I could maybe reference the `docker-compose` explicitly, but was trying to avoid doing so in order to not have outdated links pointing at the wrong lines in the file (it's standard docker patterns we are talking about here).

**Pull request merged**